### DISCUSSION TITLE
Change default root direction and invalid 403 error pages

### DISCUSSION THREAD

**Discussion by EDIflyer** [YOURLS/YOURLS#3533] [Q&A] [answered]
I'm very impressed with YOURLS - have it setup running on Docker behind NPM and works brilliantly.

I understand that navigating to `domain.tld` or to `domain.tld/invalidshortlink` both show a 403 error page **by design** but I was wondering how to change them to direct to a customised page?

**Comment by dgw** [comment, marked as answer]
Root: Add an index.(php|html) file
Invalid shortlink: This redirects to the root by default.

https://github.com/YOURLS/YOURLS/blob/fd70f6daac94925a1e98f25b065b83a822156365/yourls-go.php#L25-L27

Plugins can override the behavior for unrecognized links, e.g. [this one](https://github.com/philhagen/ltc-fuzzy-keyword-suggestions/blob/master/plugin.php) that fuzzy-suggests possible matches.

**Reply by EDIflyer** [reply]
Fantastic, thanks @dgw
